extensions/sts.md

```diff
@@ -84,7 +84,7 @@ Servers MAY send this key to all clients, but insecurely connected clients MUST
 
 Preload list providers MUST only consider hosts for inclusion after validating their connection security and ensuring a valid STS policy with a `preload` key is in place. This allows IRC network administrators to opt-in for inclusion in preload lists.
 
-Servers SHOULD be prepared to offer secure connections for the long term when enabling a preload policy. Timely removal of hostnames from preload lists might not be possible.
+Servers SHOULD be prepared to offer secure connections for the long-term when enabling a preload policy. Timely removal of hostnames from preload lists might not be possible.
```

Comment:
This is correct without the hyphen.

Reply:
I have to disagree on this but I will change it now.

Reply:
The hyphen would make this an incomplete phrase. "Long-term" is an adjective, while "long term" is a noun ("term") described by an adjective ("long").

This particular phrase is a fairly common mix-up in writing, but the presence (or absence) of a hyphen is important.